dashboard/test/ui/features/step_definitions/check_finish_button.rb

```diff
@@ -23,7 +23,7 @@
 When /^I check that the (blockly|droplet|minecraft) free play level for "([^"]*)" shows the finish button for (small|mobile) screens/i do |level_type, level_name, screen_type|
   individual_steps <<-STEPS
     And I set up the #{level_type} free play level for "#{level_name}"
-    #{screen_type == 'small' ? 'And I change the browser window size to 1366 by 600' : ''}
+    #{screen_type == 'small' ? 'And I change the browser window size to 1366 by 636' : ''}
```

Comment:
The infobar is 36 pixels according to [chromium documentation](https://www.chromium.org/user-experience/infobars/). Because of the infobar, one of the small screen tests was failing -- bumping this by 36 pixels fixed it.